docker/Dockerfile.k8s

```diff
@@ -0,0 +1,61 @@
+FROM golang:1.12
+WORKDIR /opt/pbm
+COPY . .
+RUN make install
+
+FROM registry.access.redhat.com/ubi7/ubi-minimal
+RUN microdnf update && microdnf clean all
+
+MAINTAINER Percona Development <info@percona.com>
+LABEL name="Percona Backup for MongoDB" \
```

Comment:
If you want to add LABELs please add them in right way, example you can find here https://github.com/percona/percona-docker/blob/master/percona-server-mongodb.42/Dockerfile.k8s#L12-L19